script/QEMU.sh

```diff
@@ -28,7 +29,10 @@ while true; do
       echo "  -S, --wait          Pause CPU on startup"
       echo "  -s, --serial=FILE   Output serial to FILE instead of stdout"
       echo "  -c, --coverage=FILE Output unique hit kernel mode %rip values to FILE"
+      echo "  -k, --enable-kvm    Enable hardware acceleration via KVM"
       echo "      --nographic     Don't display the graphics (default is SDL or curses)"
+      echo "      --no-reboot     Don't reboot the machine on hardware faults"
+      echo "      --no-gdb        Don't expose a gdb stub (useful if running in parallel)"
```

Comment:
This slipped by me the other time, but these `--no` flags don't follow the same convention.